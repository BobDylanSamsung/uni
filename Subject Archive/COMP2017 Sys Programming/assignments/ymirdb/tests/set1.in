set a 1 2 3
set b 4 5 6
set c 8 2 3
list entries
bye
