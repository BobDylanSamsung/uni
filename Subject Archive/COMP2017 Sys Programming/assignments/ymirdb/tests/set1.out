ok

ok

ok

c [8 2 3]
b [4 5 6]
a [1 2 3]

bye
